data/formats-data.js

```diff
@@ -4811,7 +4818,7 @@ exports.BattleFormatsData = {
 		tier: "PU",
 	},
 	archen: {
-		randomBattleMoves: ["stoneedge", "rockslide", "earthquake", "uturn", "pluck", "headsmash"],
+		randomBattleMoves: ["acrobatics", "rockslide", "earthquake", "uturn", "knockoff", "headsmash", "heatwave", "defog", "stealthrock"],
```

Comment:
archen never runs heatwave

Reply:
but it shouldnt have to worry about ev investment now